File: RepeatingElement/RepeatingElement.cs
using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        Console.Write("Enter number: ");
        int n = int.Parse(Console.ReadLine());

        Console.Write("Enter values: ");
        string[] input = Console.ReadLine().Split();
        int[] arr = Array.ConvertAll(input, int.Parse);

        Dictionary<int, int> dict = new Dictionary<int, int>();

        foreach (int num in arr)
        {
            if (dict.ContainsKey(num))
                dict[num]++;
            else
                dict[num] = 1;
        }

        Console.Write("Repeating elements are: ");
        foreach (var item in dict)
        {
            if (item.Value > 1)
                Console.Write(item.Key + " ");
        }
    }
}

File: RepeatingElement/RepeatingElement.py
# Find Repeating element in array:
n = int(input("Enter number: "))
arr = list(map(int, input("Enter values: ").split()))
d = {}
for i in arr:
    if i in d:
        d[i] += 1
    else:
        d[i] = 1

print("Repeating elements are: ", end="")
for k, v in d.items():
    if v > 1:
        print(k, end=" ")
